### ISSUE TITLE
implement possibility to increase unknown portion with training steps

### ISSUE THREAD

**Issue by ChristofHenkel** [ChristofHenkel/speech_recognition#8]
since we have far more data with label "unknown" it might help to start with a small portion for symmetry reasons, but as soon as the model starts to overfit the regular labels increase unknown to train there better
